components/alert/AlertPopulations.vue

```diff
@@ -56,17 +56,47 @@
 
       <div v-show="mode === 'number'">
         <line-chart
+          v-if="!(alert.duration > 5)"
```

Comment:
This will not work. You need to simply add a check if the data for the chart exists or not, if it doesn't display the message.